### PR TITLE
Eiko_hangman merge

### PR DESCRIPTION
**Pull request by Eiko-Tokura**
added new game hangman, fixed several typos.

### CODE REVIEW

File: osucat-reborn/osucat-main.hpp [hunk 1]
```diff
@@ -169,7 +169,7 @@ namespace osucat {
 					info(msg.substr(4), tar, params);
 					return true;
 				}
-				if (_stricmp(msg.substr(0, 1).c_str(), "i") == 0) {
+				if (_stricmp(msg.substr(0, 1).c_str(), "textinfo") == 0) {
```
Comment (Eiko-Tokura) [inline osucat-reborn/osucat-main.hpp:172]: should be msg.substr(0, 8)

Comment (Monodesu) [reply osucat-reborn/osucat-main.hpp:172]: if ((_stricmp(msg.substr(0, 1).c_str(), "i") == 0) ||  (_stricmp(msg.substr(0, 8).c_str(), "textinfo") == 0)) {
}

Comment (Eiko-Tokura) [reply osucat-reborn/osucat-main.hpp:172]: !i was changed to !textinfo to avoid conflicts with hangman.. we can change to another short command if needed

**Review by Monodesu — APPROVED** [review]
done

**Pull request merged**